README.md

```diff
@@ -4,8 +4,7 @@
 
 > One paragraph statement about the project.
```

Comment:
- [ ] Your README has a title for Portfolio whereas this is a different project, kindly correct your title
- [ ] Kindly add a more descriptive paragraph about the project

Reply:
Okay @TedLivist I will fix it!